bff-tiltaksarrangor/src/main/kotlin/no/nav/amt/tiltak/bff/tiltaksarrangor/VeilederController.kt

```diff
@@ -18,7 +23,7 @@ import org.springframework.web.bind.annotation.*
 import java.util.*
 
 @RestController("VeilederControllerTiltaksarrangor")
-@RequestMapping("/api/tiltaksarrangor/veiledere")
```

Comment:
Hvorfor flyttes denne? Konvensjon?
Hvis det skal være flere pather er det vel bedre å ha flere controllere?

Reply:
Jeg vurderte å lage ny controller, men da får vi i praksis to veileder-controllere. Da synes jeg det var bedre å gjøre denne sånn at vi kunne ha alt som gjelder veileder i en controller :) Det kan hende vi burde ha tatt en runde på hva apieene våre bør hete (hva vi ønsker å bruke som path) sånn at vi slipper denne typen flytting.